packages/atlas/src/components/_overlays/ContextMenu/ContextMenu.styles.ts

```diff
@@ -1,9 +1,42 @@
+import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-import { cVar } from '@/styles'
+import { cVar, sizes } from '@/styles'
+
+export type ContextMenuSizes = 'small' | 'medium' | 'large'
+
+type StyledContainerProps = {
+  scrollable: boolean
+  size?: ContextMenuSizes
+}
+
+const getHeightStyles = ({ scrollable, size }: StyledContainerProps) => {
+  if (!scrollable) {
+    return
+  }
+  switch (size) {
+    case 'large':
+      return css`
+        max-height: 224px;
+      `
+    case 'medium':
+      return css`
+        max-height: 188px;
+      `
+    default:
+      return css`
+        max-height: 152px;
+      `
+  }
+}
+
+export const StyledContainer = styled.div<StyledContainerProps>`
+  ${getHeightStyles};
 
-export const StyledContainer = styled.div`
   background-color: ${cVar('colorBackgroundStrong')};
-  width: 200px;
+  width: 192px;
   word-break: break-all;
+  border-radius: ${cVar('radiusSmall')};
+  overflow: auto;
+  padding: ${({ scrollable }) => sizes(scrollable ? 2 : 1)} 0;
```

Comment:
I just asked Adam about this, he made a mistake. We shouldn't change this padding based on if it's scrollable or not. This should be always 4px